controller/src/main/java/org/jboss/as/controller/services/path/PathResourceDefinition.java

```diff
@@ -134,6 +133,7 @@ public void registerAttributes(ManagementResourceRegistration resourceRegistrati
         SimpleAttributeDefinition pathAttr = specified ? PATH_SPECIFIED : PATH_NAMED;
         resourceRegistration.registerReadWriteAttribute(pathAttr, null, new PathWriteAttributeHandler(pathManager, pathAttr, services));
         resourceRegistration.registerReadOnlyAttribute(READ_ONLY, null);
+        PathSizeMetricHandler.registerMetric(resourceRegistration, pathManager, PATH, RELATIVE_TO);
```

Comment:
We need to narrow down which variants of of PathResourceDefinition we register this, as it's not meaningful in some. For example, it's not for meaningful for domain.xml path resources (NamedPathResourceDefinition) because there is no such thing as a domain-wide FS location. It's questionable for host.xml ones too; e.g. a path that is relative-to jboss.server.base.dir means nothing in the HC process; it's only when it gets converted to a server process that it means something.